test/1_stdlib/NewArray.swift.gyb

```diff
@@ -26,8 +26,7 @@ import StdlibUnittest
 // Instead of testing with Int elements, we use this wrapper class
 // that can help us track allocations and find issues with object
 // lifetime inside Array implementations.
-final class X : ForwardIndex, Comparable, CustomStringConvertible, 
-                IntegerLiteralConvertible 
+final class X : Comparable, CustomStringConvertible, IntegerLiteralConvertible 
```

Comment:
Feel free to replace this one with LifetimeTracked, too.

Also, I feel like `IntegerLiteralConvertible` decreases the readability of the test, so it would be best to remove that conformance.